src/scripts/main.js

```diff
@@ -1,7 +1,40 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+
```

Comment:
Consider adding a class or ID to the notification element for better styling and manipulation. This will help in targeting the notification more precisely with CSS or JavaScript.